src/renderer/components/MapEditor/index.js

```diff
@@ -96,6 +96,15 @@ insertCss(`
     justify-content: center;
     position: relative;
   }
+  .id-container .inspector-body .raw-tag-editor {
+    display: none;
+  }
+  .id-container .inspector-body .raw-member-editor {
+    display: none;
+  }
+  .id-container .inspector-body .raw-membership-editor {
+    display: none;
+  }  
```

Comment:
optional nitpick, but can shorten this to

```suggestion
  .id-container .inspector-body .raw-tag-editor,
  .id-container .inspector-body .raw-member-editor,
  .id-container .inspector-body .raw-membership-editor {
    display: none;
  }
```

Reply:
I personally prefer the more explicit option, it's easier to read.